backend/windmill-api/src/jobs.rs

```diff
@@ -3154,11 +3154,12 @@ async fn check_tag_available_for_workspace(
         let tags = get_scope_tags(authed);
 
         if let Some(tags) = tags {
-            if !tags.contains(&tag.as_str()) {
+            if !tags.contains(&tag.as_str()) && !authed.is_admin {
```

Comment:
it's not workspace admin it's superadmin, tags and workers are a global instance property, being a workspace admin is not sufficient